lib/main.go

```diff
@@ -356,12 +356,17 @@ func (obj *Main) Run() error {
 	}
 
 	var gapiChan chan error // stream events are nil errors
+	world := &etcd.World{
```

Comment:
This isn't even gofmt-ed. You should run `make gofmt` on your code before committing. We have a test case that catches badly formatted code.